e2e/tests-dfx/ledger.bash

```diff
@@ -60,12 +64,28 @@ teardown() {
     assert_match "1000000100.00000000 ICP"
 
     assert_command dfx ledger transfer --icp 100 --e8s 1 --memo 2 345f723e9e619934daac6ae0f4be13a7b0ba57d6a608e511a00fd0ded5866752 # to alice
-    assert_match "Transfer sent at BlockHeight:"
+    assert_match "Transfer sent at block height"
 
     # The sender(bob) paid transaction fee which is 0.0001 ICP
     # 10100 - 100 - 0.0001 - 0.00000001 = 9999.99989999
     assert_command dfx ledger balance
     assert_match "999999999.99989999 ICP"
+
+    # Transaction Deduplication
+    t=$(current_time_nanoseconds)
+
+    assert_command dfx ledger transfer --icp 1 --memo 1 --created-at-time "$t" 345f723e9e619934daac6ae0f4be13a7b0ba57d6a608e511a00fd0ded5866752
+    assert_match "Transfer sent at block height"
+
+    assert_command dfx ledger transfer --icp 1 --memo 1 --created-at-time $((t+1)) 345f723e9e619934daac6ae0f4be13a7b0ba57d6a608e511a00fd0ded5866752
+    assert_match "Transfer sent at block height"
+
+    assert_command dfx ledger transfer --icp 1 --memo 1 --created-at-time "$t" 345f723e9e619934daac6ae0f4be13a7b0ba57d6a608e511a00fd0ded5866752
+    assert_match "transaction is a duplicate of another transaction in block"
```

Comment:
```suggestion
    assert_match "transaction is a duplicate of another transaction in block"
    assert_match "Transfer sent at block height"
```

Reply:
On a duplicate transaction the output does not contain the line "Transfer sent at block height". 
Is it good to include the line "Transfer sent at block height" even on a duplicate transaction response?

Reply:
Good call, done.